### ISSUE TITLE
CosmosDBTrigger use the UseDefaultJsonSerialization parameter not correct

### ISSUE THREAD

**Issue by schw3de** [Azure/azure-webjobs-sdk-extensions#600]
#### Repro steps

Provide the steps required to reproduce the problem

Use the CosmosDBTriggerSamples in the project.

1. Step A

Set the UseDefaultJsonSerialization to true.

```
public static void Listen(
[CosmosDBTrigger("ItemDb", "ItemCollection", LeaseCollectionName = "Leases", UseDefaultJsonSerialization = true)] 
```

2. Step B

Run the sample.

#### Expected behavior

Exception should occur since no JsonConvert.DefaultSettings was set.
If you have declared the default settings those should take into account.

#### Actual behavior

The UseDefaultJsonSerialization set the parameter for useMultipleWriteLocations instead.

#### Known workarounds

Here is a commit for a possible bugfix:
https://github.com/schw3de/azure-webjobs-sdk-extensions/commit/35bb7dde63908017f18e835b51baa98b6551d654

#### Related information 

Known in:
Microsoft.Azure.WebJobs.Extensions.CosmosDB  Version=3.0.4

**Comment by schw3de** [comment]
If you want I can open a Pull Request from the fork. Just let me know.

**Comment by ealsur** [comment]
This was fixed on the linked PR and shipped in 3.0.8
